client-src/elements/chromedash-all-features-page.ts

```diff
@@ -15,37 +28,34 @@ export class ChromedashAllFeaturesPage extends LitElement {
     ];
   }
 
-  static get properties() {
-    return {
-      rawQuery: {type: Object},
-      title: {type: String},
-      showQuery: {type: Boolean},
-      query: {type: String},
-      columns: {type: String},
-      showEnterprise: {type: Boolean},
-      sortSpec: {type: String},
-      user: {type: Object},
-      start: {type: Number},
-      num: {type: Number},
-      starredFeatures: {type: Object},
-      selectedGateId: {type: Number},
-    };
-  }
+  @property({type: String})
+  title = 'Features';
+  @property({type: Boolean})
+  showQuery = true;
+  @property({type: Object})
+  user: User = null!;
```

Comment:
`null!` doesn't make much sense. Do you mean
```suggestion
  @property({type: Object})
  user!: User;
```